packages/kbn-scout-reporting/src/reporting/report/persistence/index_templates.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { IndicesPutIndexTemplateRequest } from '@elastic/elasticsearch/lib/api/types';
+import * as componentTemplates from './component_templates';
+
+export const testEvents: IndicesPutIndexTemplateRequest = {
+  name: 'scout-test-events',
```

Comment:
I see this index name hard coded in a number of places here (also as part of other bits, e.g. `dataStreamName`) . Might be a candidate for the scout-info package?